lib/WasmReader/WasmBytecodeGenerator.cpp

```diff
@@ -785,8 +840,8 @@ WasmBytecodeGenerator::GetConstReg(T constVal)
     Js::RegSlot location = m_funcInfo->GetConst(constVal);
     if (location == Js::Constants::NoRegister)
     {
-        WasmRegisterSpace * regSpace = GetRegisterSpace(m_reader->m_currentNode.type);
-        location = regSpace->AcquireConstRegister();
+        WasmRegisterSpace * regSpace = GetRegisterSpace(WasmTypes::I32);
+        location = regSpace->AcquireTmpRegister();
```

Comment:
This is not valid since tmp reg will be reused and not guaranteed to hold constant (though it looks like you aren't using this method anyway).

For background, since we don't know the count of constants up front I pivoted away from using the const table. I think all the code to GetConst, etc. can be removed, but for now maybe just revert this part.